src/mlang/m_ir/mir.ml

```diff
@@ -246,11 +246,22 @@ type variable_data = {
 (**{1 Verification conditions}*)
 
 (** Errors are first-class objects *)
+
 module Error = struct
+  module ErrorDesc = struct
+    type t = {
```

Comment:
You don't need to put it inside another module, just write `type error_descr = ...`